packages/ssr/src/types.ts

```diff
@@ -3,9 +3,9 @@ import type { CookieSerializeOptions } from 'cookie';
 export type CookieOptions = Partial<CookieSerializeOptions>;
 export type CookieOptionsWithName = { name?: string } & CookieOptions;
 export type ServerCookieMethods = {
```

Comment:
I wonder if now we should just have on set of types since the `ServerCookieMethods` and `BrowserCookieMethods` are exactly the same.

Reply:
Good call! Have consolidated them into one 👍